cmd/buildah/common.go

```diff
@@ -1,15 +1,28 @@
 package main
 
 import (
+	"encoding/json"
 	"os"
+	"strings"
+	"time"
 
 	is "github.com/containers/image/storage"
+	"github.com/containers/image/types"
 	"github.com/containers/storage"
 	"github.com/pkg/errors"
 	"github.com/projectatomic/buildah"
 	"github.com/urfave/cli"
 )
 
+type imageMetadata struct {
+	Tag            string              `json:"tag"`
+	CreatedTime    time.Time           `json:"created-time"`
+	ID             string              `json:"id"`
+	Blobs          []types.BlobInfo    `json:"blob-list"`
+	Layers         map[string][]string `json:"layers"`
+	SignatureSizes []string            `json:"signature-sizes"`
+}
```

Comment:
We're not providing anything better (yet), so this is probably going to have to do until we update the storage and image libraries to provide this information as part of their public APIs.